modules/images-editing-image-registry-settings-hcp.adoc

```diff
@@ -0,0 +1,114 @@
+// Module included in the following assemblies:
+//
+// * openshift_images/image-configuration-hcp.adoc
+// * post_installation_configuration/preparing-for-users.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="images-editing-image-registry-settings-hcp_{context}"]
+= Editing image registry settings for {hcp-title}
+
+After verifying that your image registry is enabled, you can update or edit the image registry with the `rosa edit` command.
+
+[WARNING]
+====
+When the `allowedRegistries` parameter is defined, all registries, are blocked unless explicitly listed. To prevent pod failure, a list of Red Hat registries is automatically whitelisted, as they are required by payload images within your environment. For disconnected clusters, mirror registries should be added by the user itself.
+====
+
```

Comment:
I would add also a second warning with text
```
Changing any registry related parameter will trigger a rollout across all machinepools 
(all machinepool nodes will be recreated, following pod draining from each node).
```
This is also visible below in the CLI output, but I think it should have a dedicated warning too.

Reply:
Ah it is here https://github.com/openshift/openshift-docs/pull/82728/files#diff-d5b3ac5a27787219fdda141baa9da8251b06649a266719af50d9bb717a109f73R13.
Can we have that warning in both sections? (Configuring image .... and Editing image)